spirv_hlsl.cpp

```diff
@@ -2795,9 +3160,21 @@ void CompilerHLSL::emit_hlsl_entry_point()
 	// Run the shader.
 	if (execution.model == ExecutionModelVertex ||
 	    execution.model == ExecutionModelFragment ||
-	    execution.model == ExecutionModelGLCompute)
+	    execution.model == ExecutionModelGLCompute ||
+	    execution.model == ExecutionModelMeshEXT)
 	{
-		statement(get_inner_entry_point_name(), "();");
+		SmallVector<string> arglist;
+		auto& fn = get<SPIRFunction>(ir.default_entry_point);
+		for (auto& arg : fn.arguments)
+		{
+			// Do not pass in separate images or samplers if we're remapping
```

Comment:
What's the motivation for this? Not sure how combined image samplers apply here.